content/python/concepts/math-module/terms/math-atan2/math-atan2.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: 'math.atan2()'
+Description: 'Returns the arc tangent of y/x in radians.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`math.atan2()`** [function](https://www.codecademy.com/resources/docs/python/functions) in Python takes two numeric values representing a point `(x, y)` and returns a float that represents the arc tangent of `y` divided by `x`. The resulting value is in radians and ranges from `-π` to `π`.
```

Comment:
Is there a particular reason for us to refer to a float using the full terminology rather than the name most people are familiar with? Aside from that, all other changes have been made! :)